bundles/org.openhab.core/src/main/java/org/openhab/core/internal/items/ExpireManager.java

```diff
@@ -240,13 +244,13 @@ public void receive(Event event) {
         }
         if (event instanceof ItemStateEvent) {
             ItemStateEvent isEvent = (ItemStateEvent) event;
-            processEvent(isEvent.getItemName(), isEvent.getItemState());
+            processEvent(isEvent.getItemName(), isEvent.getItemState(), true);
         } else if (event instanceof ItemCommandEvent) {
             ItemCommandEvent icEvent = (ItemCommandEvent) event;
-            processEvent(icEvent.getItemName(), icEvent.getItemCommand());
-        } else if (event instanceof GroupItemStateChangedEvent) {
```

Comment:
Is there a specific reason to remove this? I can see a use case for the expire functionality on a group (e.g. to turn off all lights in a group if they stay on for too long).